client/disputes/utils.ts

```diff
@@ -45,3 +49,8 @@ export const isDueWithin = ( { dueBy, days }: IsDueWithinProps ): boolean => {
 	const isPastDue = now.isAfter( dueByMoment );
 	return isWithinDays && ! isPastDue;
 };
+
+export const isInquiry = ( dispute: Dispute | CachedDispute ): boolean => {
+	// Inquiry disputes statuses are one of `warning_needs_response`, `warning_under_review` or `warning_closed`.
+	return dispute.status.indexOf( 'warning' ) === 0;
```

Comment:
No practical difference, but potentially a _slightly_ more readable alternative.

```suggestion
	return dispute.status.startsWith( 'warning' );
```